.github/2.0/cookbooks/Hubble.md

```diff
@@ -11,6 +11,7 @@ Table of Contents
   - [3. Use in Jina Flow](#3-use-in-jina-flow)
     - [3.2 using docker images](#32-using-docker-images)
     - [3.2 using source codes](#32-using-source-codes)
+    - [3.2 override default __init__ parameters](#32-override-parameters)
```

Comment:
TOC in all markdown are taken care by the dev bot, no need to add this manually

Reply:
ok